boards/arm/samv7/samv71-xult/configs/mcuboot-nsh/defconfig

```diff
@@ -5,81 +5,59 @@
 # You can then do "make savedefconfig" to generate a new defconfig file that includes your
 # modifications.
 #
-# CONFIG_MMCSD_MMCSUPPORT is not set
-# CONFIG_MMCSD_SPI is not set
 # CONFIG_SAMV7_UART0 is not set
 # CONFIG_SAMV7_UART2 is not set
 # CONFIG_SAMV7_UART4 is not set
 CONFIG_ARCH="arm"
-CONFIG_ARCH_BOARD_CUSTOM=y
-CONFIG_ARCH_BOARD_CUSTOM_DIR="../autococo2-board"
-CONFIG_ARCH_BOARD_CUSTOM_DIR_RELPATH=y
-CONFIG_ARCH_BOARD_CUSTOM_NAME="ARCH_BOARD_AUTOCOCO2_BOARD"
+CONFIG_ARCH_BOARD="samv71-xult"
+CONFIG_ARCH_BOARD_SAMV71_XULT=y
+CONFIG_ARCH_BUTTONS=y
 CONFIG_ARCH_CHIP="samv7"
-CONFIG_ARCH_CHIP_SAME70=y
-CONFIG_ARCH_CHIP_SAME70Q21=y
-CONFIG_ARCH_CHIP_SAME70Q=y
+CONFIG_ARCH_CHIP_SAMV71=y
+CONFIG_ARCH_CHIP_SAMV71Q21=y
+CONFIG_ARCH_CHIP_SAMV71Q=y
 CONFIG_ARCH_CHIP_SAMV7=y
+CONFIG_ARCH_CHIP_SAMV7_MEM_FLASH=0x200000
+CONFIG_ARCH_CHIP_SAMV7_MEM_RAM=0x60000
 CONFIG_ARCH_INTERRUPTSTACK=2048
+CONFIG_ARCH_IRQBUTTONS=y
 CONFIG_ARCH_STACKDUMP=y
 CONFIG_ARMV7M_DCACHE=y
 CONFIG_ARMV7M_ICACHE=y
 CONFIG_ARMV7M_LAZYFPU=y
-CONFIG_AT24XX_ADDR=0x57
-CONFIG_AT24XX_EXTENDED=y
-CONFIG_AT24XX_EXTSIZE=160
-CONFIG_AT24XX_SIZE=2
+CONFIG_BOARDCTL_RESET=y
 CONFIG_BOARD_LATE_INITIALIZE=y
 CONFIG_BOARD_LOOPSPERMSEC=51262
 CONFIG_BOOT_MCUBOOT=y
 CONFIG_BUILTIN=y
-CONFIG_DEBUG_FULLOPT=y
-CONFIG_DEBUG_SYMBOLS=y
-CONFIG_FAT_LCNAMES=y
-CONFIG_FAT_LFN=y
-CONFIG_FS_FAT=y
 CONFIG_FS_PROCFS=y
-CONFIG_I2CTOOL_MAXBUS=0
-CONFIG_INTELHEX_BINARY=y
 CONFIG_MCUBOOT_SLOT_CONFIRM_EXAMPLE=y
-CONFIG_MCUBOOT_VERSION="1a9c6d8495e4dbe7d02edf14bb8a9fa1d4e955c0"
-CONFIG_MMCSD_MULTIBLOCK_DISABLE=y
-CONFIG_MMCSD_SDIO=y
-CONFIG_MTD=y
-CONFIG_MTD_AT24XX=y
-CONFIG_MTD_AT25=y
```

Comment:
Why this configuration is not included?

Reply:
It not seems to be relevant in my view. Is there any particular purpose to add those?

Reply:
No. That is fine. Just was thinking `mcuboot-nsh` as an extension of `nsh` with MCUboot capabilities, so was curious why this config is different from `nsh` one